rama-tls-acme/src/proto/server.rs

```diff
@@ -0,0 +1,464 @@
+use serde::{Deserialize, Serialize};
+
+use super::common::Identifier;
+
+pub const REPLAY_NONCE_HEADER: &str = "replay-nonce";
+pub const LOCATION_HEADER: &str = "location";
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+/// Directy contain all endpoints needed by an acme client, defined in [rfc8555 section 7.1.1]
+///
+/// [rfc8555 section 7.1.1]: https://datatracker.ietf.org/doc/html/rfc8555/#section-7.1.1
+pub struct Directory {
+    pub new_nonce: String,
```

Comment:
document these fields please, and in general for these very technical bits it is very welcome do not only have summary line but also extended docs